admin-dev/themes/new-theme/js/components/form-submit-button.js

```diff
@@ -60,11 +60,32 @@ export default class FormSubmitButton {
         return;
       }
 
+      let method = 'POST';
```

Comment:
Great addition to this component!! Can you update the JSDoc on the class and add the new `data-method` parameter please?